lib/autoproj/workspace.rb

```diff
@@ -803,6 +803,19 @@ def export_env_sh(_package_names = nil, shell_helpers: true)
             full_env = self.full_env
             changed = save_cached_env(full_env)
             full_env.export_env_sh(shell_helpers: shell_helpers)
+
+            build_dir_is_absolute = File.absolute_path(build_dir) == build_dir
```

Comment:
I think `Pathname#absolute?` is more robust

Reply:
Not sure it's more robust, but it would definitely look better. I'll change.

Reply:
> Not sure it's more robust

```ruby
File.absolute_path("/root/////") != "/root/////"
```